frontend/src/pages/CollaborationPage.js

```diff
@@ -57,9 +57,7 @@ const CollaborationPage = () => {
 
   const getQuestion = async () => {
     try {
-      const res = await axios.get(
-        'http://localhost:8100/api/question?difficulty=easy'
-      )
+      const res = await findQuestion(location.state.difficulty)
```

Comment:
actually since the try catch is here is also ok hor? if this is the case, then the previous comments are resolved